charts/stable/mosquitto/values.yaml

```diff
@@ -9,17 +9,32 @@ service:
       main:
         port: 1883
         targetPort: 1883
+  websockets:
+    enabled: '{{ .Values.websockets.enabled }}'
```

Comment:
There is no support to `tpl` in this key.
So it will always evaluate to true (strings with any value are truthy)

You can leave it `enabled: true` and only inject the config (as you do in the configmap already) if websockets enabled

Reply:
ok thanks - done